pyuvdata/uvcal/uvcal.py

```diff
@@ -5348,14 +5513,53 @@ def read(
                 # Because self was at the beginning of the list,
                 # everything is merged into it at the end of this loop
         else:
+            if file_type in ["fhd", "calfits"]:
+                if (
+                    antenna_nums is not None
+                    or antenna_names is not None
+                    or frequencies is not None
+                    or freq_chans is not None
+                    or spws is not None
+                    or times is not None
+                    or lsts is not None
+                    or time_range is not None
+                    or lst_range is not None
+                    or jones is not None
+                ):
+                    select = True
+                    warnings.warn(
```

Comment:
hmm. Ok. Let me check with some HERA folks, who were the ones pushing for select on read, so presumably have some trouble with the size. It may be because they often process entire nights at once.

Reply:
I don't think it's a bit issue to keep this in, so I'll withdraw my comment for now, though perhaps we can re-evaluate if users indicate it to be a nuisance.